frontend/test/metabase/visualizations/lib/settings/visualization.unit.spec.js

```diff
@@ -59,29 +59,74 @@ describe("visualization_settings", () => {
         ));
     });
   });
+  describe("getDefault", () => {
```

Comment:
Seems like 

```
describe("table.cell_column", () => {
  describe("getDefault", () => {
```

would make more sense (or just remove the `getDefault` describe entirely)